specs/schemas/workspace.json

```diff
@@ -1,10 +1,14 @@
 {
-  "$schema": "<http://json-schema.org/draft-07/schema#>",
+  "$schema": "http://json-schema.org/draft-07/schema",
   "type": "object",
   "properties": {
     "version": {
       "type": "string"
     },
+    "kiotaVersion": {
```

Comment:
actually I'd remove it. The design was to store the version in the associated API manifest.